package.json

```diff
@@ -5,7 +5,8 @@
   "main": "server.js",
   "scripts": {
     "start": "node server.js",
-    "dev": "nodemon --watch src server.js"
+    "dev": "nodemon --watch src server.js dev",
+    "prod": "nodemon --watch src server.js prod"
```

Comment:
in produzione sarebbe meglio non utilizzare `nodemon`, per il momento utilizziamo `node nomefile.js....`, in seguito bisognerebbe utilizzare un runner di produzione come descritto in #20